app/gateway/2.8.x/admin-api/developers/reference.md

```diff
@@ -0,0 +1,1417 @@
+HTTP/1.1 200 OK
+```
+
+```json
+{
+    "error": {
+        "count": 0,
+        "emails": {}
+    },
+    "sent": {
+        "count": 2,
+        "emails": {
+            "example@example.com": true,
+            "example2@example.com": true
+        }
+    },
+    "smtp_mock": true
+}
+```
+
+### Inspect a developer
+
+**Endpoint**
+
+<div class="endpoint get">/developers/{DEVELOPER_EMAIL|DEVELOPER_ID}</div>
+
+
+Attribute                     | Description
+---------:                    | --------   
+`{DEVELOPER_EMAIL|DEVELOPER_ID}`  | The email or UUID of the developer you want to inspect.
+
+
+**Response**
+
+```
+HTTP/1.1 200 OK
+```
+
+```json
+{
+    "consumer": {
+        "id": "855fe098-3bb3-4041-8a3d-149a9e5462c6"
+    },
+    "created_at": 1644617754,
+    "email": "example@example.com",
+    "id": "f94e4fbd-c30b-41d7-b349-13c8f4fc23ca",
+    "rbac_user": {
+        "id": "7d40540b-4b65-4fa6-b04e-ffe6c2c40b0d"
+    },
+    "roles": [
+        "QA"
+    ],
+    "status": 0,
+    "updated_at": 1644619493
+}
+```
+
+### Update a developer
+
+**Endpoint**
+
+<div class="endpoint patch">/developers/{DEVELOPER_EMAIL|DEVELOPER_ID}</div>
+
+Attribute                     | Description
+---------:                    | --------   
+`{DEVELOPER_EMAIL|DEVELOPER_ID}`  | The email or UUID of the developer you want to update.
+
+
+**Request Body**
+
+Attribute                     | Description
+---------:                    | --------   
+`meta`                        | Metadata for the account in JSON format. Accepts fields defined in the Dev Portal settings. <br><br> By default, the meta attribute requires a `full_name` field. You can remove this requirement, or add other fields as neccessary. <br><br> For example: `meta: {"full_name":"<NAME>"}`.
+`email`       | The email of the developer to create. This becomes their login username.
+
+
+**Response**
+
+```
+HTTP/1.1 200 OK
+```
+
+```json
+{
+    "developer": {
+        "consumer": {
+            "id": "855fe098-3bb3-4041-8a3d-149a9e5462c6"
+        },
+        "created_at": 1644617754,
+        "email": "example@example.com",
+        "id": "f94e4fbd-c30b-41d7-b349-13c8f4fc23ca",
+        "rbac_user": {
+            "id": "7d40540b-4b65-4fa6-b04e-ffe6c2c40b0d"
+        },
+        "roles": [
+            "QA"
+        ],
+        "status": 0,
+        "updated_at": 1644621148
+    }
+}
+```
+
+### Delete a developer
+
+**Endpoint**
+
+<div class="endpoint delete">/developers/{DEVELOPER_ID}</div>
+
+Attribute                     | Description
+---------:                    | --------   
+`{DEVELOPER_ID}`  | The UUID of the developer you want to delete.
+
+**Response**
+
+```
+HTTP/1.1 204 No Content
+```
+
+## Export developer metadata
+
+Export a list of developers and their metadata into a CSV file.
+
+Creates a file named `developers.csv` in the present working directory.
+
+<div class="endpoint get">/developers/export</div>
+
+**Response**
+
+```
+HTTP/1.1 200 OK
+```
+
+```json
+something
```

Comment:
Need an example for the response for `/developers/export`. I don't have a 2.8 preview build, so I can't test this.

Reply:
The format of the response is like this:
```
Email, Status
test@example.com,APPROVED
test1@example.com,PENDING
test2@example.com,REVOKED
test3@example.com,REJECTED
```